### ISSUE TITLE
Ex: Folder name 2.5 ordered before 2

### ISSUE THREAD

**Issue by begleysm** [UniversalMediaServer/UniversalMediaServer#1858]
It is common for TV shows to have each season in a dedicated folder (ie: Season 1, Season 2, Season 3, etc).  Occasionally there is a need to describe an intermediate season (ie: Season 2.5).

UMS defaults to ordering Season 2.5 prior to Season 2 in the web GUI.  I think this should be made to order Season 2.5 after Season 2.  See screenshot below showing the Folder Structure on the right that resulted in a web GUI ordering on the left.
![sorting](https://user-images.githubusercontent.com/1275999/66865081-8219ae80-ef64-11e9-986a-f4c13b9aef11.png)

**Comment by SubJunk** [comment]
@begleysm thanks, that's a good catch. Have you tried changing the sorting option to see if that gives the expected behavior? 
![image](https://user-images.githubusercontent.com/584228/74635950-ea56f600-51cb-11ea-8117-e35c76e3f498.png)

**Comment by begleysm** [comment]
I started this issue Oct 15 2019.  I'm currently running 9.1.0 which came out on Nov 01 2019.  I just went back to try to reproduce the issue prior to messing around with the Sorting option and was unable to do so.  Perhaps it was fixed in 9.1.0.

I'm running UMS under Debian and so I don't have the GUI you show above.  However I went into UMS.conf and am currently under default sort_method (which is 4 which is Alphanumeric)

See screenshot below for folder ordering in 9.1.0
![image](https://user-images.githubusercontent.com/1275999/74669961-370fe200-5176-11ea-8236-3cfc011c36d5.png)

Given the above I suppose this issue can be closed as resolved.
